src/import-data/import-utils.feature-flag.test.ts

```diff
@@ -0,0 +1,23 @@
+import { ImportRequest } from './import-types';
+import { getCloudPlatformRequiredForImport } from './import-utils';
+
+jest.mock('src/libs/feature-previews', () => ({
+  ...jest.requireActual('src/libs/feature-previews'),
+  isFeaturePreviewEnabled: jest.fn().mockReturnValue(true),
+}));
+
+// Note that behavior of getRequiredCloudPlatformForImport when the feature flag is set to false is tested
+// in import-utils.test.ts. This file only tests behavior when the feature flag is set to true.
+// Since `jest.mock` is global for any given file, the test below is given this new file.
```

Comment:
Not asking for changes, but for what it's worth, you _can_ alter the behavior of `isFeaturePreviewEnabled` for different tests within a file with something like:

```ts
import { isFeaturePreviewEnabled } from 'src/libs/feature-previews';

jest.mock('src/libs/feature-previews', () => {
  return {
    ...jest.requireActual('src/libs/feature-previews'),
    isFeaturePreviewEnabled: jest.fn().mockReturnValue(true),
  };
});

describe('with feature flag enabled', () => {
  beforeEach(() => {
    asMockedFn(isFeaturePreviewEnabled).mockReturnValue(true);
  });

  it(...)
});

describe('with feature flag disabled', () => {
  beforeEach(() => {
    asMockedFn(isFeaturePreviewEnabled).mockReturnValue(true);
  });

  it(...);
});
```

Reply:
ah nice, thank you!